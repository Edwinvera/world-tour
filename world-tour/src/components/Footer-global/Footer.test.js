import renderer from 'react-test-renderer'
import 'jest-styled-components'
import React from 'react'
import Footer from './Footer'

test('Footer component renders', () => {
  const FooterComponent = renderer.create(<Footer/>).toJSON()
  expect(FooterComponent).toMatchSnapshot()
})