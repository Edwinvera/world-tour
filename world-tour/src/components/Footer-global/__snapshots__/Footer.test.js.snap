// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders 1`] = `
.c0 {
  position: absolute;
  bottom: 0px;
  width: 100%;
  height: .1vh;
  text-align: center;
  opacity: .9;
}

.c1 {
  background-color: rgb(28,28,28);
  padding: 1px 0;
}

.c2 {
  font-family: 'Merriweather Sans',sans-serif;
  font-size: 16px;
  color: #DAE2DF;
}

<div>
  <footer
    className="c0"
  >
    <div
      className="c1"
    >
      <h6
        className="c2"
      >
        Â© Stefon Simmons
      </h6>
    </div>
  </footer>
</div>
`;
